internal/utils/token.go

```diff
@@ -31,3 +33,35 @@ func GenerateJWT(user *models.UserResponse) (string, error) {
 
 	return tokenString, err
 }
+
+func GetUserId(r *http.Request) string {
+	if r.Header["Authorization"] != nil {
+
+		authHeader := strings.Split(r.Header.Get("Authorization"), "Bearer ")
+
+		if len(authHeader) != 2 {
+			ErrorLogger.Println("Malformed Token")
+			return ""
+		} else {
```

Comment:
You can write it without else. You have return in previous block

Reply:
Fixed: Rewritten without else-block